test/system/250-systemd.bats

```diff
@@ -174,10 +174,14 @@ function check_listen_env() {
     if is_remote; then
 	is "$output" "$stdenv" "LISTEN Environment did not pass: $context"
     else
-	is "$output" "$stdenv
+	out=$(for o in $output; do echo $o; done| sort)
```

Comment:
If for any reason you need to resubmit, could I suggest this instead?
```
    out=$(sort <<<"$output")
```
Reason: if there's ever an environment variable whose value includes a space (unlikely), the `for` loop will print each space-separated token on a new line.